packages/cicero-core/src/templatelibrary.js

```diff
@@ -246,6 +266,19 @@ class TemplateLibrary {
         return `${this.url}/${prefix}template-library.json`;
     }
 
+    /**
+     * @returns {object} the httpOptions of Template Library
+     */
+    getHttpOptions() {
+        return {
+            uri: `${this.url}/template-library.json`,
+            headers: {
+                'User-Agent': 'clause',
+                Authorization: this.httpHeader,
+            },
+            json: true, // Automatically parses the JSON string in the response
+        };
+    }
```

Comment:
@mttrbrts The HTTPOptions were getting complicated since there was no source to return those auto created HTTPOptions so I created to get these Options.